http/http-advanced/src/test/java/io/quarkus/ts/http/advanced/BaseHttpAdvancedIT.java

```diff
@@ -244,7 +244,7 @@ private ResponsePredicateResult isHttp2x(HttpResponse<Void> resp) {
     private WebClientOptions defaultVertxHttpClientOptions() {
         return new WebClientOptions().setProtocolVersion(HttpVersion.HTTP_2).setSsl(true).setVerifyHost(false)
                 .setUseAlpn(true)
-                .setTrustStoreOptions(new JksOptions().setPassword(PASSWORD).setPath(defaultTruststore()));
+                .setTrustAll(true);
```

Comment:
Why this? How does it solve the problem with keycloak memory?

Reply:
Nope, it's not related, I made a mistake.